doc/lightningd-config.5.md

```diff
@@ -309,6 +309,14 @@ up space in the database.
 Control how long invoices must have been expired before they are cleaned
 (if *autocleaninvoice-cycle* is non-zero).
 
+Payment control options:
+
+ **disable-mpp**=*BOOL*
+Disable the multi-part payment sending support in the `pay` plugin. By default
+the MPP support is enab led, but it can be desirable to disable in situations
```

Comment:
'enab led'....

Reply:
I really hate the double-negative.  Should be just "**disable-mpp**", with no BOOL option?  Or *enable-mpp* with default true...